openedx/adg/lms/webinars/helpers.py

```diff
@@ -305,3 +316,39 @@ def cancel_all_reminders(registrations, is_rescheduling=False):
     WebinarRegistration.objects.bulk_update(
         registrations, ['starting_soon_mandrill_reminder_id', 'week_before_mandrill_reminder_id'], batch_size=999
     )
+
+
+def get_webinar_invitees_emails(webinar_form):
+    """
+    Given a webinar form, get emails of all invitees of the webinar
+
+    Arguments:
+        webinar_form (Form): Model form with updated data.
+
+    Returns:
+        list: Webinar invitees' emails
+    """
+    webinar_invitees_emails = webinar_form.cleaned_data.get('invites_by_email_address', [])
+    if webinar_form.cleaned_data.get('invite_all_platform_users'):
+        webinar_invitees_emails += list(
+            User.objects.exclude(email='').values_list('email', flat=True)
```

Comment:
Why do we need to exclude empty email addresses? Are there users without any email?

Reply:
In the User object provided by Django, email is not a compulsory field, so we can have users created from the admin site without providing an email address.